### PR TITLE
A minor revision of agented-vm-guilde.md

### PR DESCRIPTION
**Pull request by jianjuns**
Change section title "ExternalEntities CR Creation" to ExternalEntity CR Creation".

Signed-off-by: Jianjun Shen <jshen.yn@gmail.com>

### CODE REVIEW

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

File: docs/agented-vm-guide.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 - [Import Agented And Agentless VMs](#import-agented-and-agentless-vms)
 - [VirtualMachine CR Creation](#virtualmachine-cr-creation)
 - [ExternalNode CR Creation](#externalnode-cr-creation)
-- [ExternalEntities CR Creation](#externalentities-cr-creation)
+- [ExternalEntity CR Creation](#externalentities-cr-creation)
```
Comment (Anandkumar26) [inline docs/agented-vm-guide.md:9]: This needs to be changed

**Review by Anandkumar26 — CHANGES_REQUESTED** [review]
TOC needs to be updated.

File: docs/agented-vm-guide.md (hunk 1 above)
Comment (jianjuns) [reply docs/agented-vm-guide.md:9]: I already changed it?

Comment (Anandkumar26) [inline docs/agented-vm-guide.md:9]: The below change was missed.
```suggestion
- [ExternalEntity CR Creation](#externalentity-cr-creation)
```

Comment (jianjuns) [reply docs/agented-vm-guide.md:9]: Ah, got it! Thanks for pointing it out!

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

**Pull request merged**